lib/Github/Api/GitData/References.php

```diff
@@ -25,6 +25,22 @@ public function all($username, $repository)
         return $this->get('/repos/'.rawurlencode($username).'/'.rawurlencode($repository).'/git/refs');
     }
 
+    /**
+     * Get all matching references for the supplied reference name
```

Comment:
@nickpoulos this is the line where styleci "complains" about. It should end with a "."